readme.md

```diff
@@ -56,6 +56,15 @@ Returns an unsubscribe method.
 
 Using the same listener multiple times for the same event will result in only one method call per emitted event.
 
+##### Custom subscribable events
+
+Emittery exports some symbols which represent custom events that can be passed to `Emitter.on` and similar functions.
+
+- `Emittery.listenerAdded` - Fires when an event listener was added. An object with `eventName` and `listener` is provided as data where possible.
+- `Emittery.listenerRemoved` - Fires when an event listener was removed. An object with `eventName` and `listener` is provided as data where possible.
```

Comment:
"is provided as data where possible" - `listener` is always there, and `eventName` is only not there in case on `onAny`/`offAny`. Need clear docs on both cases.

Reply:
Fixed!

Reply:
Still no docs on what happens when `onAny`/`offAny` is used.

Reply:
@stroncium Fixed!